src/frontend/js/components/Tabs/_styles.scss

```diff
@@ -0,0 +1,42 @@
+$tabs-scheme: map-get($themes, 'default', 'components', 'tabs');
+$selected-color: map-get($themes, 'default', 'color', 'primary-600');
```

Comment:
By trying locally, I feel this variable as no value. What do you think to define this color through the tabs-scheme ?